### PR TITLE
feat: add link to logion identity LOC.

### PR DESCRIPTION
**Pull request by gdethier**
logion-network/logion-internal#299

### CODE REVIEW

File: src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx [hunk 1]
```diff
@@ -157,8 +169,25 @@ export default function ContextualizedLocDetails() {
 
                                 <Col md={ 4 } className="closed-icon-container">
                                     <LocItemDetail
-                                        label="Requested by">{ locRequest.userIdentity?.firstName || "" } { locRequest.userIdentity?.lastName || "" }<br />
-                                        { locRequest.requesterAddress || "-" }
+                                        label="Requested by"
+                                    >
+                                        { locRequest.userIdentity?.firstName || "" } { locRequest.userIdentity?.lastName || "" }
+                                        {
+                                            locRequest.requesterAddress !== null && locRequest.requesterAddress !== undefined &&
```
Comment (benoitdevos) [inline src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx:176]: I think this shorter line does the same:
```suggestion
                                            locRequest.requesterAddress &&
```

File: src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx [hunk 2]
```diff
@@ -157,8 +169,25 @@ export default function ContextualizedLocDetails() {
...
+                                            <span><br /> { locRequest.requesterAddress }</span>
+                                        }
+                                        {
+                                            locRequest.requesterIdentityLoc !== null && locRequest.requesterIdentityLoc !== undefined &&
```
Comment (benoitdevos) [inline src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx:180]: Same remark:
```suggestion
                                            locRequest.requesterIdentityLoc &&
```

**Review by benoitdevos — APPROVED** [review]
Approved with small suggestion to shorten the code.

File: src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx (hunk 1 above)
Comment (gdethier) [reply src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx:176]: Unfortunately, this does not work here: JSX expects a boolean, not a "falsy" expression.

File: src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx (hunk 2 above)
Comment (gdethier) [reply src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx:180]: Same answer.

**Pull request merged**